test/Microsoft.Framework.Logging.Test/TraceSourceScopeTest.cs

```diff
@@ -6,7 +6,6 @@
 #if DNX451
```

Comment:
This file doesn't even use Moq... can we remove this `using` and get rid of all the `#if`s in this file?